synapse/lib/storm.py

```diff
@@ -782,6 +782,38 @@
             $lib.print("Disabled cron job: {iden}", iden=$iden)
         ''',
     },
+    {
+        'name': 'ps',
```

Comment:
We should create a namespace for the commands and a Lib namespace in storm.  ps.list / ps.kill and $lib.ps.list() etc probably work well.

Reply:
Moved to ps namespace.